ipykernel/jsonutil.py

```diff
@@ -45,6 +45,9 @@
 JPEG = b'\xff\xd8'
 # front of JPEG base64-encoded
 JPEG64 = b'/9'
+# constants for identifying gif data
+GIF_64 = b'R0lGODdh'
+GIF89_64 = b'R0lGODlh'
```

Comment:
@takluyver As long as this looks good to you, then I think that this PR is ready to go!